lib/preload.js

```diff
@@ -251,11 +265,19 @@ preload.CONTAINER_NAME = 'resin-image-preloader'
 
 /**
  * Build the preloader docker image
+ * @param {Object} docker - dockerode instance
+ * @param {Object} visuals - resin-cli-visuals (optional)
  * @returns {Stream}
  */
-preload.build = (docker) => {
+preload.build = (docker, visuals) => {
```

Comment:
The standalone mode has no visuals.
It is deprecated and won't get any new feature (like visuals).
I'm just keeping it functional until `preload` through `resin-cli` is well tested.
I'll probably remove it next month.

I'll make the changes to require `resin-cli-visuals` in resin-preload.